pkg/cmd/server/admin/create_kubeconfig.go

```diff
@@ -183,6 +186,7 @@ func (o CreateKubeConfigOptions) CreateKubeConfig() (*clientcmdapi.Config, error
 		CurrentContext: o.ContextNick,
 	}
 
+	fmt.Fprintf(o.GetOut(), "Generating '%s' API client config as %s\n", o.UserNick, o.KubeConfigFile)
```

Comment:
To differentiate generically between "the file existed and I blew it away" vs "the file wasn't there so I created it" would be *another* boolean to pass up the call stack... complicated by there being two files in most cases... no, I don't really want to do that. The user ran the command with or without --overwrite, they will probably have the context to understand the output.